bokehjs/src/coffee/models/widgets/selectbox.ts

```diff
@@ -22,9 +22,9 @@ export class SelectView extends InputWidgetView {
   build_options(values): HTMLElement[] {
     return values.map((el) => {
       if (isString(el))
-        value = _label  = el
+        const value = _label = el
       else
-        [value, _label] = el
+        const [value, _label] = el
```

Comment:
This is incorrect. `value` and `_label` are local to `if-else` branches with this setup. You need to declare them with `let` before `if-else`. Given that this is a partially typed *.ts, use `gulp dev-build --include widgets/selectbox` to see all compilation errors (by default only syntactical and a few other are shown).

Reply:
Thanks, that's really helpful! I still have a lot to learn about ES6 and TypeScript.